package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "data-api",
+    "version": "0.0.1",
+    "description": "Node.js applicaiton that performs configured experiments on data ingested by hdd scouts and enable users verify(or disprove) their hypothesis.",
+    "dependencies": {
+        "express": "^4.16.2",
+        "validator": "^9.2.0"
+    },
+    "engine": {
+        "node": "*",
+        "npm": "*"
+    },
+    "scripts": {
+        "test": "mocha --timeout 5000 tests/*_test.js"
```

Comment:
You can add linter related script here as in https://github.com/fabric8-hdd/manager-api/pull/2/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R6
So that a developer can execute that script to check for linter errors and also we can actually have linter jobs on this kind of libraries also